File: Plugins/Aspose_Barcode_Java_for_PHP/src/aspose/barcode/WorkingWithBarcode/AdvanceBarcodeFeatures/StartStopSymbol.php
<?php
namespace Aspose\Barcode\WorkingWithBarcode\AdvanceBarcodeFeatures;

use com\aspose\barcode\BarCodeBuilder as BarCodeBuilder;
use com\aspose\barcode\Symbology as Symbology;
use com\aspose\barcode\CodabarSymbol as CodabarSymbol;

class StartStopSymbol{

    public static function run($dataDir=null){

        # Create instance of BarCodeBuilder, specify codetext and symbology in the constructor
        $symbology=new Symbology();
        $builder = new BarCodeBuilder("$ 123:50", $symbology->Codabar);

        # Set the codabar start symbol to A
        $codabarSymbol=new CodabarSymbol();
        $builder->setCodabarStartSymbol($codabarSymbol->A);

        # Set the codabar stop symbol to D
        $builder->setCodabarStopSymbol($codabarSymbol->D);

        # Save the image to disk in PNG format
        $builder->save($dataDir . "barcode.out.png");

        # Print message
        print "Barcode image generated successfully.".PHP_EOL;

    }


}

File: Plugins/Aspose.BarCode Java for Jython/asposebarcode/WorkingWithBarcode/AdvanceFeatures/StartStopSymbol.py
from asposebarcode import Settings
from com.aspose.barcode import BarCodeBuilder
from com.aspose.barcode import Symbology
from com.aspose.barcode import CodabarSymbol

class StartStopSymbol:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithBarcode/AdvanceBarcodeFeatures/StartStopSymbol/'
        
        # Create instance of BarCodeBuilder, specify codetext and symbology in the constructor
        symbology=Symbology
        builder = BarCodeBuilder(" 123:50", symbology.Codabar)

        # Set the codabar start symbol to A
        codabarSymbol = CodabarSymbol
        builder.setCodabarStartSymbol(codabarSymbol.A)

        # Set the codabar stop symbol to D
        builder.setCodabarStopSymbol(codabarSymbol.D)

        # Save the image to disk in PNG format
        builder.save(dataDir + "barcode.out.png")

        # Print message
        print "Barcode image generated successfully."

if __name__ == '__main__':        
    StartStopSymbol()